src/pages/messages/components/MessageList.tsx

```diff
@@ -36,7 +67,12 @@ export const TicketMessageList = ({ ticket }: MessageListProps) => {
             text={message.text}
             date={message.createdDate}
             username={message.sender}
+            canDeleteMessage={user && (canDeleteMessage || user.nvaUsername === message.sender)}
             backgroundColor={ticketColor[ticket.type]}
+            onDelete={() => deleteSupportMessageMutation.mutateAsync(message.identifier)}
+            isDeleting={deleteSupportMessageMutation.isPending}
```

Comment:
Noen grunn til at mutationen blir laget her og sendt nedover?

Reply:
`MessageList` håndterer kun `messages`, mens det på en `nviCandidate` er en `note`. Disse har forskjellige endepunkt for sletting, derfor sendes den nedover i `MessageList` 👍